unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Transform/DCLTransform.cs

```diff
@@ -67,5 +70,61 @@ public void UpdateFromModel(BaseModel model)
         public BaseModel GetModel() => DCLTransform.model;
         public int GetClassId() => (int) CLASS_ID_COMPONENT.TRANSFORM;
         public void UpdateOutOfBoundariesState(bool enable) { }
+        
+        public static void CapTransformGlobalValuesToMax(Transform transform)
```

Comment:
is it necessary to be static? could be an instance method? on the other hand it could be refactor into an extension method: `public static void ClampToMaxGlobalValues(this Transform transform)` so the call would cleaner: `transform.ClampToMaxGlobalValues()`

Reply:
Good idea! I've moved that method [into Utils](https://github.com/decentraland/unity-renderer/blob/f84254771fc59f64c5aca9791a4b2e032e203b85/unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/Utils/Utils.cs#L93) next to the commonly used [`ResetLocalTRS()`](https://github.com/decentraland/unity-renderer/blob/f84254771fc59f64c5aca9791a4b2e032e203b85/unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/Utils/Utils.cs#L149)